README.adoc

```diff
@@ -127,12 +140,15 @@ followed by fields that correspond to metadata fields.
 *The `description` field is optional and describes the purpose of the metric
 annotation.
 
-The `@Counted` annotation applies to the `list()` method. This annotation counts how many times you request the list of systems methods by visiting `http://localhost:9080/inventory/systems` to view the current contents of the
-inventory.
-* The `absolute` field makes the name of this annotation take the name of the method.
+The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
```

Comment:
There should be a blank line before this for the list to properly show up.

Reply:
Done